docs/multiarch.rst

```diff
@@ -523,6 +524,51 @@ the API method ``create_prod_build`` specifying which platform a build
 is required for. If no platform is specified, no node selector will be
 used.
 
+Platform description
```

Comment:
Ideally, the osbs configuration file would already be YAML and we'd add a new key alongside 'instances'.

I think it makes sense to have a single configuration file for this. After all, platform names are used elsewhere in this configuration file, so it's useful to have a nearby reference for what each platform is.